node/libs/roles/src/proto/validator/messages.proto

```diff
@@ -39,57 +39,69 @@ message Block {
 message View {
   reserved 1,2;
   reserved "protocol_version","fork";
+
   optional GenesisHash genesis = 4; // required
   optional uint64 number = 3; // required; ViewNumber
 }
 
 message ConsensusMsg {
-  oneof t {// required
-    ReplicaPrepare replica_prepare = 1;
+  reserved 1, 3, 4;
+  reserved "replica_prepare", "leader_prepare", "leader_commit";
+
+  oneof t { // required
     ReplicaCommit replica_commit = 2;
-    LeaderPrepare leader_prepare = 3;
-    LeaderCommit leader_commit = 4;
+    ReplicaTimeout replica_timeout = 5;
```

Comment:
Please use a separate message type for chonkybft messages (i.e. sth like ConsensusMsgV2, or ChonkyMsg).

Reply:
Why?